Dockerfile.foo

```diff
@@ -1,19 +1,19 @@
-FROM --platform=${BUILDPLATFORM} golang:1.24.3-alpine3.21 as builder
+FROM --platform=${BUILDPLATFORM} golang:1.24.3-alpine3.21 AS builder
 
 WORKDIR /workspace
 
 COPY go.mod go.sum /workspace/
 RUN go mod download
 
 COPY cmd /workspace/cmd
-COPY private /workspace/private
+COPY internal /workspace/internal
 
 ARG TARGETOS
 ARG TARGETARCH
 RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} \
   go build -ldflags "-s -w" -trimpath -o /go/bin/foo ./cmd/foo
 
-FROM --platform=${TARGETPLATFORM} alpine:3.21.3
```

Comment:
- RedundantTargetPlatform: Setting platform to predefined ${TARGETPLATFORM} in FROM is redundant as this is the default behavior (line 16)